packages/http/src/utils/fakeIncomingMessage.ts

```diff
@@ -0,0 +1,112 @@
+/* eslint-disable prettier/prettier */
```

Comment:
Sadly, we can't disable the formatting for a block of code, only a single line or a whole file.  Allowing all the unimplemented getters and setters at the end of the class to be compactly formatted makes it far easier to see what's going on than when they're spread out over 50+ lines.  I'd rather have the type checking @daniel-white suggested than the code format enforcement in this case (since I can only have 2 out of 3).